_posts/en/newsletters/2023-09-06-newsletter.md

```diff
@@ -0,0 +1,129 @@
+      include a signature plus a public key that is typically 33 bytes,
+      the public key is omitted and a technique of reconstructing it
+      from signature is used.
+
+    Some other techniques are used to save a few extra bytes in typical
+    transactions.  The general downside of the proposal is that
+    converting a compressed transaction back into something that full
+    nodes and other software can use requires more CPU, memory, and I/O
+    than processing a regular serialized transaction.  That means
+    high-bandwidth connections will likely continue to use the regular
+    transaction format and only low-bandwidth transmission will use
+    compressed transactions.
+
+    The idea received a moderate amount of discussion, mostly around
+    ideas for saving a small amount of additional space per input.
+
+- **Privacy enhanced co-signing:** Nick Fallow [posts][fallow cosign] to
+  the Bitcoin-Dev mailing list about how a [scriptless threshold
+  signature scheme][topic threshold signature] like [FROST][] could
+  improve the privacy of people who use co-signing services.  A typical
+  user of a co-signing service has multiple signing keys that are stored
+  separately for security; but, to simplify normal spending, they also
+  allow their outputs to spent by a combination of some of their keys
+  plus one or more keys held by one or more service providers who only
+  sign after authenticating the user in some way.  The user can bypass
+  the service provider if they need to, but the service provider makes
+  operations easier in most cases.
+
+    With scripted threshold signature schemes like 2-of-3
+    `OP_CHECKMULTISIG`, the service's public key must be associated with
+    the output being spent, so any service will be able to find the
+    transactions it signed by looking at onchain data, allowing it
+    accumulate data about its users.  Worse, all currently used protocols
+    we're aware of directly reveal user transactions to the service
+    provider before signing, allowing the service to refuse to sign
+    certain transactions.
+
+    As Farrow describes, FROST allows hiding the signed transaction from
+    the service at every step of the process, from generation of an
+    output script, to signing, to publication of the fully signed
+    transaction.  All the service will know is when it signed and any
+    data the user provided to authenticate themselves with the service.
+
+    The idea received some discussion on the mailing list.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test release candidates.*
+
+- [LND v0.17.0-beta.rc1][] is a release candidate for the next major
+  version of this popular LN node implementation.  A major new
+  experimental feature planned for this release, which could likely
+  benefit from testing, is support for "simple taproot channels".
```

Comment:
```suggestion
  benefit from testing, is support for _simple taproot channels_.
```

Reply:
Leaving as-is.  "Simple taproot channels" is a LND name for a set of features that aren't yet part of the protocol and I don't think we want to treat that name as normative until we know it has buy-in from other implementations, especially since there's been debate about the LND approach in the past (as I understand it).